staff_features/shared/step_definitions.rb

```diff
@@ -53,10 +87,6 @@
   fill_in label, with: @uuid, match: :first
 end
 
-When 'the user fills in {string} with {string}' do |label, value|
-  fill_in label, with: value, match: :first
-end
```

Comment:
Is there a reason this was removed? It looks like it is still used in a few places.

Reply:
Fixed. It was accidentally removed. Note, that from now on, cucumber is running with `--strict` flag enabled, so tests are failing in advance if there is a missing step definition.